docs/pages/concepts/why-liveblocks.mdx

```diff
@@ -0,0 +1,118 @@
+---
+meta:
+  title: "Why Liveblocks"
+  parentTitle: "Concepts"
+  description:
+    "Liveblocks is a real-time collaboration infrastructure for building
+    performant collaborative experiences."
+---
+
+Everything we do at Liveblocks comes from the strong belief that flexible office
+policies are here to stay, and as a result, every SaaS product will eventually
+be real-time collaborative to help people feel connected with one another and
+work better together.
```

Comment:
```suggestion
Everything we undertake at Liveblocks derives from the firm belief that flexible office
policies are here to stay, and that as a result, every SaaS product will eventually
be real-time collaborative, helping people feel more connected with one another and
work better together.
```

I've carefully written an alternative version too, in case you prefer it:

> At Liveblocks, we firmly believe that flexible office policies are here to stay, and that as 
> a result, all SaaS products will eventually use real-time collaboration. Every project
> we undertake at Liveblocks stems from this belief, and the belief that we can empower
> people to work better together, and feel more closely connected with one another.

Also an extra-confident alternative:

> At Liveblocks, we're certain that flexible office policies are here to stay. Every
> SaaS product will eventually need real-time collaboration, empowering more 
> effective work, and closer connections between people.

I really thought about these, I think the first paragraph is important.

Reply:
Love those, much much much better. Important to nail that paragraph since it's the first thing people will read.